DigitalLearningSolutions.Data/DataServices/TicketDataService.cs

```diff
@@ -5,7 +5,9 @@
 
     public interface ISupportTicketDataService
     {
-        public int GetNumberOfUnarchivedTicketsForCentreId(int centreId);
+        int GetNumberOfUnarchivedTicketsForCentreId(int centreId);
+
+        int GetNumberOfUnarchivedTicketsForAdminId(int adminId);
     }
 
     public class SupportTicketDataService : ISupportTicketDataService

```

Comment:
Not a proper review, but I spotted this while looking at something else:
The file name is "TicketDataService" but the class name is "SupportTicketDataService" and it would be good to have them consistent (probably both "SupportTicketDataService"